### PR TITLE
Create 10K_PDF_Summary notebook

### PR DESCRIPTION
**Pull request by ZHIHANCHEN03**
Create new notebook to display the process of using huggingface flow generate context and summary pair

### CODE REVIEW

File: examples/10K_Evaluator/10K_PDF_Summary.ipynb [hunk 1]
```diff
@@ -0,0 +1,745 @@
...
+    "\n",
+    "- Create a conda environment specific to this project. Follow the provided setup instructions for guidance.\n",
+    "\n",
+    "#### Installation of Dependencies\n",
+    "\n",
+    "Install necessary libraries within your environment:\n",
+    "\n",
+    "```bash\n",
+    "pip3 install uniflow\n",
+    "pip3 install \"pykoi[huggingface, rag, rlhf]\"\n",
```
Comment (goldmermaid) [inline examples/10K_Evaluator/10K_PDF_Summary.ipynb]: remove pykoi

File: examples/10K_Evaluator/10K_PDF_Summary.ipynb [hunk 2]
```diff
@@ -0,0 +1,745 @@
...
+   "outputs": [],
+   "source": [
+    "%reload_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "import sys\n",
+    "\n",
+    "sys.path.append(\".\")\n",
+    "sys.path.append(\"..\")\n",
+    "sys.path.append(\"../..\")"
```
Comment (goldmermaid) [inline examples/10K_Evaluator/10K_PDF_Summary.ipynb]: let's delete the system paths cell

Comment (goldmermaid) [reply examples/10K_Evaluator/10K_PDF_Summary.ipynb]: in the next cell, add `!{sys.executable} -m pip install -q uniflow==0.0.23` (i.e. we install uniflow directly from pip)

File: examples/10K_Evaluator/10K_PDF_Summary.ipynb [hunk 3]
```diff
@@ -0,0 +1,745 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ba8692ce-094f-4bb5-afa7-3dba85944b7d",
+   "metadata": {},
+   "source": [
+    "# Build Your Own 10K Agent: Transform a Unstructured Financial Report to an Finetuned LLM"
```
Comment (goldmermaid) [inline examples/10K_Evaluator/10K_PDF_Summary.ipynb]: Remove the misleading title

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**